xml/obs_scm_ci_workflow_integration.xml

```diff
@@ -0,0 +1,1044 @@
+        pull request is created, somebody pushes a commit, a pull request is
+        merged, etc. When you set up a webhook on the SCM side, you can
+        specify which events you are interested in. Only when those events
+        happen, the webhook will be sent to OBS.</para>
+
+        <para>This is the list of SCM events supported by the existing
+        workflows in OBS:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para>Pull requests</para>
+          </listitem>
+
+          <listitem>
+            <para>Pushes</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>Other SCMs like GitLab have similar behaviour; the main
+        difference is how they name the events. Refer to the <link
+        linkend="sec.obs.obs_scm_ci_workflow_integration.setup.equivalence_table">Equivalence
+        Table</link> for more details or read more about <link
+        xlink:href="https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads">GitHub
+        events</link> and <link
+        xlink:href="https://docs.gitlab.com/ce/user/project/integrations/webhooks.html#events">GitLab
+        events</link>.</para>
+      </sect3>
+    </sect2>
+
+    <sect2 xml:id="sec.obs.obs_scm_ci_workflow_integration.setup.obs_workflows">
+      <title>OBS Workflows</title>
+
+      <para>An SCM event occurs and OBS receives the corresponding webhook.
+      Now is when the <emphasis role="bold">OBS workflows</emphasis> come into
+      play.</para>
+
+      <para>A workflow is nothing else than a sequence of <emphasis
+      role="bold">steps</emphasis> or actions the user wants to perform in
+      OBS. The user can describe the steps to run in a YAML configuration
+      file.</para>
+
+      <para>To do so, in the root directory of your GitHub repository, create
+      a directory <emphasis>.obs</emphasis> which contains a file called
+      <emphasis>workflows.yml</emphasis>.</para>
+
+      <para>The content of <emphasis>.obs/workflows.yml</emphasis> could look
+      like this:<screen>main_workflow:
+  steps:
+    - branch_package
+        source_project: OBS:Server:Unstable
+        source_package: obs-server
+        target_project: OBS:Server:Unstable:CI
+  filters:
+    event: pull_request
+rebuild_master:
+  steps:
+    - rebuild_package
+        project: home:Admin
+        package: ctris
+  filters:
+    event: push
+    branches:
+      only:
+        - master</screen></para>
+
+      <para>The file can contain multiple workflows, but each one needs an
+      <emphasis>unique</emphasis> name. The previous example contains two
+      workflows: <emphasis>main_workflow</emphasis> and
+      <emphasis>rebuild_master</emphasis>.</para>
+
+      <para>OBS assumes the configuration file is placed in your target branch
+      and not in other branches or forks.</para>
+
+      <note>
+        <title>Workflows Cleanup</title>
```

Comment:
If everything just works and people can not do anything about this, do we even have to mention it?

Reply:
Good point, removing